pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2559,6 +2562,8 @@ public CompletableFuture<? extends TopicStatsImpl> asyncGetStats(GetStatsOptions
         stats.waitingPublishers = getWaitingProducersCount();
         stats.bytesOutCounter = bytesOutFromRemovedSubscriptions.longValue();
         stats.msgOutCounter = msgOutFromRemovedSubscriptions.longValue();
+        stats.bytesOutInternalCounter = bytesOutFromRemovedSystemSubscriptions.longValue();
```

Comment:
What about the values from an active (not removed) subscription? Aren't you missing them?

Reply:
No, seems not missed.
This line just add removed subscriptions stats, active system subscriptions' stats added: https://github.com/apache/pulsar/blob/352c316bd4c1f85b546ad70271feff78eae8f78a/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java#L2595-L2596